test/test_relational_sklearn.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2019 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from sklearn.preprocessing import MinMaxScaler as SkMinMaxScaler
+
+from lale.datasets.multitable.fetch_datasets import fetch_go_sales_dataset
+from lale.lib.rasl import MinMaxScaler as RaslMinMaxScaler
+
+
+class TestMinMaxScaler(unittest.TestCase):
```

Comment:
Maybe add one more test that uses RaslMinMaxScaler in a pipeline with LogisticRegression?

https://github.com/IBM/lale/blob/0ac28084a0a13b1749ebce471fa483cf4b31c90d/test/test_core_transformers.py#L82-L88

Reply:
I was not able to have the Spark pipeline working because the output of `MinMaxScaler` must be converted to Pandas to be a valid input to `LogisticRegression` (see commit 75b795b).
I didn't see an existing operator to do that.

Should I merge the PR without this test and add a `Convert(to="pandas")` operator in another PR?

Reply:
See commit 3891b47